src/main/java/io/gravitee/reporter/elasticsearch/mapping/es7/ES7IndexPreparer.java

```diff
@@ -54,7 +57,14 @@ protected Function<Type, CompletableSource> indexTypeMapper() {
 
             final String template = freeMarkerComponent.generateFromTemplate("/es7x/mapping/index-template-" + typeName + ".ftl", data);
 
-            return client.putTemplate(templateName, template);
+            final Completable templateCreationCompletable = client.putTemplate(templateName, template);
+            if (configuration.isIlmManagedIndex()) {
+                return templateCreationCompletable
+                    .andThen(client.getAlias(aliasName))
+                    .ignoreElement()
+                    .onErrorResumeNext(throwable -> client.createIndexAndAlias(aliasName + "-000001", aliasName));
```

Comment:
createIndexAndAlias must be called only if alias can't be found. 
Here, the createIndexAndAlias can be done in case of IO problems with ES, which is not correct.